erizo/src/erizo/rtp/QualityFilterHandler.cpp

```diff
@@ -0,0 +1,130 @@
+#include "rtp/QualityFilterHandler.h"
+
+#include "./WebRtcConnection.h"
+#include "lib/ClockUtils.h"
+#include "rtp/RtpUtils.h"
+
+namespace erizo {
+
+DEFINE_LOGGER(QualityFilterHandler, "rtp.QualityFilterHandler");
+
+QualityFilterHandler::QualityFilterHandler()
+  : connection_{nullptr}, enabled_{true}, initialized_{false},
+  target_spatial_layer_{0}, target_temporal_layer_{0},
+  video_sink_ssrc_{0}, video_source_ssrc_{0}, last_ssrc_received_{0} {}
+
+void QualityFilterHandler::enable() {
+  enabled_ = true;
+}
+
+void QualityFilterHandler::disable() {
+  enabled_ = false;
+}
+
+void QualityFilterHandler::handleFeedbackPackets(std::shared_ptr<dataPacket> packet) {
+  RtpUtils::forEachRRBlock(packet, [this](RtcpHeader *chead) {
+    RtpUtils::updateREMB(chead, 400000);
```

Comment:
Why 400000? Even if we have proper feedback soon, it probably would make more sense to use `maxVideoBW` here